projects/pgai/db/tests/vectorizer/test_vectorizer.py

```diff
@@ -2229,3 +2235,28 @@ def test_weird_primary_key():
             cur.execute("select ai.vectorizer_queue_pending(%s)", (vectorizer_id,))
             actual = cur.fetchone()[0]
             assert actual == 7
+
+
+def test_install_ai_extension_before_library():
+    with psycopg.connect(db_url("test")) as con:
+        with con.cursor() as cur:
+            cur.execute("drop schema if exists ai cascade")
+            cur.execute("create extension ai cascade")
+
+    import pgai
+
+    pgai.install(db_url("test"))
```

Comment:
This test can only run from inside the docker container. It doesn't work on the host, which makes debugging an issue.